vs-code-extensions.md

```diff
@@ -0,0 +1,226 @@
+    'react-hooks/rules-of-hooks': 'error',
+    'react-hooks/exhaustive-deps': 'warn',
+    'react-native/no-unused-styles': 2,
+    'react-native/split-platform-components': 2,
+    'react-native/no-single-element-style-arrays': 2,
+    'react-native/no-inline-styles': 'off',
+    'prettier/prettier': 'error',
+    'jsx-quotes': ['error', 'prefer-single'],
+    'no-shadow': 'off',
+    'import/prefer-default-export': 'off',
+    'react/no-unescaped-entities': 'off',
+    'prefer-destructuring': [
+      'error',
+      {
+        array: false,
+        object: true,
+      },
+    ],
+    'max-lines': ['error', { max: 500, skipComments: true }],
+  },
+  ignorePatterns: ['node_modules/'],
+};
+```
+
+<br/>
+<br/>
+
+## `React Native Tools`.
+This VSCode extension provides a development environment for React Native projects. Using this extension, you can debug your code and quickly run react-native commands from the command palette.
+
+VSCode Market Place: [React Native Tools](https://marketplace.visualstudio.com/items?itemName=msjsdiag.vscode-react-native)
+
+Installation:
+```
+ext install msjsdiag.vscode-react-native
+```
+
+<br/>
+<br/>
+
+## `Code Spell Checker`.
+A basic spell checker that works well with camelCase code.
+
+VSCode Market Place: [Code Spell Checker](https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker)
+
+Installation:
+```
+ext install streetsidesoftware.code-spell-checker
+```
+
+<br/>
+<br/>
+
+## `Command Runner`.
+Run custom shell command defined in vs code configuration and node module package.json
+
+We can use it to quickly run different emulators.
+
+VSCode Market Place: [Command Runner](https://marketplace.visualstudio.com/items?itemName=edonet.vscode-command-runner)
+
+Installation:
+```
+ext install edonet.vscode-command-runner
+```
+VSCode settings configuration:
+```json
+{
+    "command-runner.terminal.name": "react-native",
+    "command-runner.terminal.autoClear": true,
+    "command-runner.terminal.autoFocus": true,
+    "command-runner.commands": {
+        "iPhone 11 Pro": "yarn react-native run-ios --scheme 'Plai Dev' --configuration 'Debug' --simulator='iPhone 11 Pro (13.5)'",
+        "iPhone 11 Pro (Prod)": "yarn react-native run-ios --scheme 'Plai' --configuration 'Release' --simulator='iPhone 11 Pro'",
+        "iPhone SE": "yarn react-native run-ios --scheme 'Plai Dev' --configuration 'Debug' --simulator='iPhone SE (2nd generation)'",
+        "iPhone 7 (real device)": "yarn react-native run-ios --scheme 'Plai Dev' --configuration 'Debug' --device='iPhone (Your_Phone_Name)'",
+        "Android": "yarn react-native run-android",
+    }
+}
+```
+<br/>
+<br/>
+
+## `Auto Rename Tag`.
+Automatically rename paired tag.
+
+VSCode Market Place: [Auto Rename Tag](https://marketplace.visualstudio.com/items?itemName=formulahendry.auto-rename-tag)
+
+Installation:
+```
+ext install formulahendry.auto-rename-tag
+```
+
+<br/>
+<br/>
+
+## `vscode-styled-components`.
```

Comment:
I think we can remove this one from the guide for now and add it back if we decide to use styled-components on all projects.